Book/php7/build_system/building_php.rst

```diff
@@ -443,8 +443,8 @@ If the ``make`` command finishes successfully, it will print a message encouragi
 
 ``make test`` will run the PHP CLI binary against our test suite, which is located in the different *tests/* directories
 of the PHP source tree. As a default build is run against approximately 9000 tests (less for a minimal build, more if
-you enable additional extensions) this can take several minutes. The ``make test`` command is currently not parallel, so
-specifying the ``-jN`` option will not make it faster.
+you enable additional extensions) this can take several minutes. The ``make test`` is also parallel, so
+specifying the ``-jN`` option will make it faster.
```

Comment:
`make test` with `-jN` has no effect, as mentioned by Nikita. You should document the env `TEST_PHP_ARGS`, as described in https://github.com/php/php-src#testing-php-source-code :)